ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/SchemaInfo.java

```diff
@@ -48,6 +66,7 @@ public SchemaInfo(
         ? null
         : ImmutableList.copyOf(fields);
     this.memberSchema = memberSchema;
+    this.fieldProperties = ImmutableMap.of();
```

Comment:
nit: if we have two constructors, it's better to have one call the other:
```java
this(type, fields, memberSchema, ImmutableMap.of();
```
That way the two won't ever get out of sync